src/sentry/middleware/auth.py

```diff
@@ -121,6 +121,9 @@ def process_request(self, request: Request):
         else:
             request.user = AnonymousUser()
 
+    def process_exception(self, request: Request, exception: Exception):
+        pass
```

Comment:
Should we be trapping exceptions here into sentry?

Reply:
I don't, think so.  We have separate django middleware that handles actually caching exceptions.  `process_exception` apparently only *possibly* catches the exception if it returns a Response object, otherwise the default is to bubble up.